interop-rx2/src/main/kotlin/com/bumble/appyx/interop/rx2/plugin/DisposeOnDestroyPlugin.kt

```diff
@@ -0,0 +1,17 @@
+package com.bumble.appyx.interop.rx2.plugin
+
+import com.bumble.appyx.core.plugin.Destroyable
+import com.bumble.appyx.core.plugin.Plugin
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+
+private class DisposeOnDestroyPlugin(disposables: List<Disposable>) : Destroyable {
+    private val disposable = CompositeDisposable(disposables)
+
+    override fun destroy() {
+        disposable.dispose()
+    }
+}
+
+fun disposeOnDestroyPlugin(vararg disposables: Disposable): Plugin =
```

Comment:
I deliberately didn't call it `disposables` like in RIBs because I feel that the extension function needs more context - especially since it isn't scoped.